privacyidea/lib/utils.py

```diff
@@ -1219,3 +1219,32 @@ def check_pin_policy(pin, policy):
                 comment.append("Missing character in PIN: {0!s}".format(chars[c]))
 
     return ret, ",".join(comment)
+
+
+def get_module_class(package_name, class_name, check_method=None):
+    """
+    helper method to load the Module class from a given
+    package in literal.
+
+    :param package_name: literal of the Module
+    :param class_name: Name of the class in the module
+    :param check_method: Name of the method to check, if this would be the right class
+
+    example:
+
+        get_module_class("privacyidea.lib.auditmodules.sqlaudit", "Audit", "log")
+
+        get_module_class("privacyidea.lib.monitoringmodules.sqlstats", "Monitoring")
+
+    check:
+        checks, if the method exists
+        if not an error is thrown
+
+    """
+    mod = __import__(package_name, globals(), locals(), [class_name])
```

Comment:
We could check if using ``importlib`` instead of ``__import__`` has any advantages: https://docs.python.org/2/library/importlib.html

But we don't have to do that in this PR, I can create an issue for that.

Reply:
Another Issue would make sense.
We are using ``__import__`` at a lot of other locations. So we should not change this *here* but have a common way to do this.

Reply:
I've opened #1335.

Reply:
:+1: